client/my-sites/customer-home/cards/features/site-preview/style.scss

```diff
@@ -11,6 +11,8 @@
 		max-height: 235px;
 		cursor: pointer;
 		transition: all 200ms ease-in-out;
+		border: 1px solid #eee;
```

Comment:
Need to use a color variable here rather than a hard-coded value, I believe it's `var( --color-border-subtle )`